liquid

```diff
@@ -1,19 +1,4 @@
 #!/bin/bash
 set -e
 
-# cd to file
-cd "$(dirname ${BASH_SOURCE[0]})"
```

Comment:
Please keep this line, since you're invoking `./liquid.py` assuming CWD is the project folder.